scripts/ci/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: mono-llvm
+Priority: optional
+Maintainer: Alex Kyte <alkyte@microsoft.com>
+Build-Depends: debhelper (>= 8.0.0)
```

Comment:
As in debian/compat note, this should be lowered to `debhelper (>= 7.0.50~)` for now.